apps/api-extractor/src/schemas/api-extractor-defaults.json

```diff
@@ -83,6 +85,5 @@
       }
     }
   },
-
```

Comment:
Should this newline be removed?

Reply:
I see newlines between other top level declarations, will palce back.